dialogflow_task_executive/package.xml

```diff
@@ -19,9 +19,11 @@
   <build_depend>catkin_virtualenv</build_depend>
   <build_depend>message_generation</build_depend>
   <build_depend>std_msgs</build_depend>
+  <build_depend>roslaunch</build_depend>
```

Comment:
why this is bulid_depend? you want to add run_depend?

Reply:
I followed this instruction

https://wiki.ros.org/roslaunch#Catkin